packages/components/src/Table/Sorters/Sorter.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * This defines the sorting direction.
+ * NONE is used a default value, it has no effect on elements order.
+ */
+export const Order = {
+	NONE: 'none',
+	ASCENDING: 'ascending',
+	DESCENDING: 'descending',
+};
+
+const coefs = {
+	none: 0,
+	ascending: 1,
+	descending: -1,
+};
+
+/**
+ * This class represents a sorter for the elements of a table.
+ * A sorter is identified by a unique string identifier.
+ */
+export default class Sorter {
```

Comment:
you already have a global state (prefered way in redux application) a local state in react component, i don't see why we need to spread this state and mutate it in uncontrolled ways outside those, more complicated to test, more error prone, more complex to maintain, use pure function and apply application state (preferred) or component state to them.